notebooks/official/tensorboard/tensorboard_profiler_custom_training_with_prebuilt_container.ipynb

```diff
@@ -130,185 +125,132 @@
-      "execution_count": null,
-      "metadata": {
-        "id": "Se9FWWhLotvB"
-      },
-      "outputs": [],
-      "source": [
-        "REGION = \"us-central1\"  # @param {type:\"string\"}"
-      ]
-    },
-    {
-      "attachments": {},
       "cell_type": "markdown",
       "metadata": {
-        "id": "IfJRIMBpo5Pg"
+        "id": "dmWOrTJ3gx13"
       },
       "source": [
-        "### Authenticate your Google Cloud account\n",
-        "\n",
-        "Depending on your Jupyter environment, you may have to manually authenticate. Follow the relevant instructions below.\n",
+        "### Authenticate your notebook environment (Colab only)\n",
         "\n",
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing as you are already authenticated.\n",
-        "\n",
-        "**2. Local JupyterLab instance, uncomment and run:**"
+        "Authenticate your environment on Google Colab.\n"
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "cR_MzpknpGgM"
+        "id": "NyKGtVQjgx13"
       },
       "outputs": [],
       "source": [
-        "# ! gcloud auth login"
+        "import sys\n",
+        "\n",
+        "if \"google.colab\" in sys.modules:\n",
+        "\n",
+        "    from google.colab import auth\n",
+        "\n",
+        "    auth.authenticate_user()"
       ]
     },
     {
-      "attachments": {},
       "cell_type": "markdown",
       "metadata": {
-        "id": "h-MuVI_ypJfw"
+        "id": "DF4l8DTdWgPY"
       },
       "source": [
-        "**3. Colab, uncomment and run:**"
+        "### Set Google Cloud project information\n",
+        "\n",
+        "To get started using Vertex AI, you must have an existing Google Cloud project. Learn more about [setting up a project and a development environment](https://cloud.google.com/vertex-ai/docs/start/cloud-environment)."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "BeaQlCwMpQUT"
+        "id": "Nqwi-5ufWp_B"
       },
       "outputs": [],
       "source": [
-        "# from google.colab import auth\n",
-        "# auth.authenticate_user()"
+        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "LOCATION = \"us-central1\"  # @param {type:\"string\"}"
       ]
     },
     {
-      "attachments": {},
       "cell_type": "markdown",
       "metadata": {
         "id": "3ivZkPUjpaFz"
       },
       "source": [
-        "**4. Setup service account and permissions**\n",
+        "**Setup service account and permissions**\n",
         "\n",
         "A service account will be used to create custom training jobs. If you do not want to use your project's Compute Engine service account, set SERVICE_ACCOUNT to another service account ID. You can create a service account by following the [instructions](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating)."
```

Comment:
change "will be" to "is":

- A service account is used to create custom training jobs